docs/installation/configuration.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Configuration Options"
+weight: 20
+---
+
+This page focuses on the options that can be used when you set up K3s for the first time:
+
+- [K3s Install Script](#configuration-with-install-script)
+- [K3s Binary](#configuration-with-binary)
+- [Configuration File](#configuration-file)
+
+For more advanced options, refer to [this page](/advanced).
+
+## Configuration with install script
+
+As mentioned in the [Quick-Start Guide](/quick-start), you can use the installation script available at https://get.k3s.io to install K3s as a service on systemd and openrc based systems.
+
+You can use a combination of `INSTALL_K3S_EXEC`, `K3S_` enviromental variables and command flags to configure the installation.
+
+To illustrate this, the following commands all result in the same behavior of registering a server without flannel and with a token:
+
+```bash
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--flannel-backend none --token 12345" sh -s -
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --flannel-backend none" K3S_TOKEN=12345 sh -s -
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server" sh -s - --flannel-backend none
+curl -sfL https://get.k3s.io | K3S_TOKEN=12345 sh -s - server --flannel-backend none
+curl -sfL https://get.k3s.io | sh -s - --flannel-backend none --token 12345
+```
+
+For details on all enviromental variables, see [Enviromental Variables.](/reference/env-variables)
+
+## Configuing with binary
+
+As stated, the installation script is primarily concerned with configuring K3s to run as a service.  
+If you choose to not use the script, you can run K3s simply by downloading the binary from our [release page](https://github.com/rancher/k3s/releases/latest), placing it on your path, and executing it. Or you can install K3s without enabling it as a service:
+```bash
+curl -sfL https://get.k3s.io | INSTALL_K3S_SKIP_ENABLE=true sh -
+```
+
+You can configure K3s in this manner through `K3S_` enviromental variables:
+```bash
+$ K3S_KUBECONFIG_MODE="644" k3s server
```

Comment:
We haven't been always consistent with placing (or not placing) the `$` in the commands, but generally, we avoid adding them in as users should be able to copy-paste directly into their shell with as few modifications as possible, and they may include these and receive errors.